GOVERNANCE.md

```diff
@@ -173,21 +173,30 @@ Each voter can vote zero or more times, once per each candidate. As this is not
   * Signatures majority of council members to validate correctness and accuracy
 * Core contributors can also voluntarily leave the project by notifying the community through a public means or by notifying the entire council.
 
-#### One time Decisions
+#### One Time Decisions
 * The first election will also include a vote for the number of Council Members, options being 5 and 7. A simple majority will indicate preference.
   * At the conclusion of the vote this section should be removed
 
-
-#### Voting criteria for future elections
+#### Voting Criteria For Future Elections
 Voting for first election is restricted to establish stable governance, and to defer major decision to elected leaders
 * For the first election only the folks in Slack can vote (excluding GSOC students)
 * In the first year, the council must determine voting eligibility for future elections between two criteria:
   * Those with commit bits
   * The contributing community at large
 
+### Core Contributors
+Core Contributors are those who have provided consistent and meaningful contributions to ArviZ.
+These can be, but are not limited to, code contributions, community contributions, tutorial 
+development etc. 
+
+#### Core Contributor Nominations and Confirmation Process
+Current Core Contributors can nominate candidates for consideration by the council. The council
+can make the determination for acceptance with a process of their choosing.
```

Comment:
- This seems vaguer than necessary -- I think we have more informative priors than this 😉 
I mean: the Council will probably accept proposed Core Contributors through the same system of election as the one used to elect / reject the Council, right? So maybe let's just say this here.
- Also, I think we never talked about that: are there any upper and lower bounds on the number of Core Contributors?

Reply:
There should be no election for core contributors to join as such, otherwise the strain on core devs and contributors would be too significant. One of the goals is to keep governance as little work as possible.

Reply:
I would not set a maximum for core contributors

Reply:
But how are Core Contributors accepted then? Informal votes?
A simple yes/no vote as is done for the first round of Council elections doesn't seem too big of a strain to me, especially if you do it in batch -- e.g every quarter end, all nominated candidates are voted in or out by the Council. Looks more straightforward and transparent to me than doing it gradually and informally 🤷‍♂️ 
No need to be as formal as for the Council though (with a PR and so on) -- a vote on Slack should be sufficient.

Reply:
Current core contributors propose people to join as core contributors and council decides as they see fit (I think not being specific on selection criteria and so on is intended in order to allow any kind of contributor to become core, not only people sending prs).

Regarding voting and so on, the normal decision rules for council would be applied (are they defined above? ).

Reply:
> I think not being specific on selection criteria and so on is intended in order to allow any kind of contributor to become core, not only people sending prs.

Completely agree with that -- was just wondering about the voting process.

Reply:
I would agree there is no voting process. None of the other projects Ive seen have had one, Even python. Someone is just like Hey Alex/Oriol etc is great lets give them a commit bit or w/e.

That and I think the council should make the decision based on the factors at the time, not a rigid process setup now. Thats my 2 cents

Reply:
Ow ok! I think we're trying to maximize different utility functions, but that makes sense too and I'm not opposed to it 👌